app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -146,18 +146,18 @@ class RecentlyPlayedFragmentTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
-  private fun createRecentlyPlayedActivityIntent(internalProfileId: Int): Intent {
+  private fun createRecentlyPlayedActivityIntent(profileId: Int): Intent {
```

Comment:
`profileId`, is a complete model class that we use for profiles and `internalProfileId` is an integer for a profile, as an index. So, here we are passing an integer so we can keep the `internalProfileId`. 
Also, thanks for removing the non required named arguments.

Reply:
done